app/react-native/src/preview/components/OnDeviceUI/index.js

```diff
@@ -1,24 +1,116 @@
-import React, { PropTypes } from 'react';
-import { View } from 'react-native';
+import React, { Component, PropTypes } from 'react';
+import { Animated, Easing, View, TouchableWithoutFeedback, Image, Text } from 'react-native';
 import style from './style';
 import StoryListView from '../StoryListView';
 import StoryView from '../StoryView';
 
-export default function OnDeviceUI(props) {
-  const { stories, events, url } = props;
+export default class OnDeviceUI extends Component {
+  constructor(props, ...args) {
+    super(props, ...args);
```

Comment:
Super nitty feedback - but since `props` isn't used, we could just spread all the arguments

```js
constructor(...props) {
  super(...props)
  ...
}
```